cli/dcoscli/data/help/marathon.txt

```diff
@@ -28,6 +28,7 @@ Usage:
     dcos marathon group show [--group-version=<group-version>] <group-id>
     dcos marathon group remove [--force] <group-id>
     dcos marathon group update [--force] <group-id> [<properties>...]
+    dcos marathon plugins [--json]
```

Comment:
By analogy with the other marathon subcommands, where it is a singular noun followed by an action, should this rather be `dcos marathon plugin list [--json]`? That'd make it easier to add plulgin related subommands in the future while keeping backward compatibility.

Reply:
makes sense to me!